cds_dojson/schemas/records/videos/project/project_src-v1.0.0.json

```diff
@@ -5,8 +5,12 @@
     {
       "properties": {
         "title": { "$ref": "../../definitions-v1.0.0.json#/definitions/title" },
```

Comment:
minor, ocd related: some keys are not formatted like the others

Reply:
I guess you are referring to the fact that all first elements of arrays should start from in a new line indented, which I also do prefer, what we ended up with here is the automagic formatting of prettier. This is the [conversation](https://github.com/prettier/prettier/issues/2716) looking into the specific configuration.